nodejs/README.md

```diff
@@ -0,0 +1,77 @@
+# Stan TV NodeJS Coding Challenge
+
+> Welcome to the Stan TV NodeJS Coding Challenge!
+
+## Overview
+
+The purpose of this challenge is to assess your knowledge around composing a **simple websocket based client/server application** closely following the guidelines and requirements.
+
+## Guidelines
+
+- The "Client" Application should be built using [Babel](https://www.babeljs.io) into a `dist/client` folder containing three files (`app.js`, `logo.svg` and `index.html`).
+- The server application should be built using [Babel](https://www.babeljs.io) into a `dist/server` folder containing a single file (`server.js`).
+- Your solution should be implemented using:
+  - [TypeScript](https://www.typescriptlang.org)
+  - [Babel](https://babeljs.io)
+  - [Jest](https://jestjs.io)
+- You should avoid using any other JS frameworks/libraries.
+- We are looking for solutions that are **simple, modern, performant, and tested**.
+
+## Requirements
+
+You will need to build the following 2 applications:
+
+### "Client" Application
+
+[client-1.jpg](./client-1.jpg)
+[client-2.jpg](./client-2.jpg)
+
+- Render a black document with the stan logo `logo.svg` horizontally and vertically centered.
+- Establish a connection to the WebSocket server.
+- The "Client" App should be available from port 8080.
+- Receive messages from the "Server" Application containing an id and url.
```

Comment:
Maybe mention the image stuff here because its not really mentioned.

`Receive messages from the "Server" Application containing an id and an image url`